releases/README.md

```diff
@@ -9,5 +9,6 @@ directory structure
   |-- ...
 ```
 
-For each release, please run command "docs/build_doc.sh" under Gearpump repository, rename `latest` directory to name reflecting previous version number
-and copy the whole `docs/site` directory (from gearpump repo, after building) to release and rename `site` directory `latest`. 
+For each release, please run the command "docs/build_doc.sh" under Gearpump repository, 
+rename `latest` directory to a name reflecting previous version number 
+and copy the whole `docs/site` directory (from Gearpump repo, after having it built) to release and rename `site` directory to `latest`.
```

Comment:
Yes, we should. Actually, there is a lot of confusion with the dependencies needed for building the docs and the commands themselves.

I am starting to experiment with dockers for building gearpump and the docs: https://github.com/karol-brejna-i/gearpump-building-docker

These would encapsulate the dependencies, the commands, etc.